### PR TITLE
Pod pid limit

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Follow-up from https://github.com/kubernetes/website/pull/13006

Had got lost in a sea of github notifications.

This is for 1.14.

### CODE REVIEW

File: docs/tasks/administer-cluster/pids.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+{{% capture body %}}
+
+## Pod to Pod Isolation of Pids
+
+The `SupportPodPidsLimit` feature gate is *beta*.
+
+If enabled, the `kubelet` argument for `pod-max-pids` will write out the configured
+pid limit to the pod level cgroup to the value specified on Linux hosts. If -1,
+the `kubelet` will default to the node allocatable pid capacity.
```
Comment (xiangpengzhao) [inline docs/tasks/administer-cluster/pids.md]: Do we need to add some words to explicitly show that the default value of `pod-max-pids` is `-1`?

File: docs/tasks/administer-cluster/pids.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+addition, it is important to ensure that pids are limited among pods in order to
+ensure they have limited impact to other workloads on the node.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Pod to Pod Isolation of Pids
+
+The `SupportPodPidsLimit` feature gate is *beta*.
```
Comment (sftim) [inline docs/tasks/administer-cluster/pids.md]: Can we make use of a `feature-state` [shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state) here?

Comment (PatrickLang) [reply docs/tasks/administer-cluster/pids.md]: And also mention that it's enabled by default in *beta*?

**Comment by DanyC97** [comment]
/language en

@derekmahar any chance you can address the comments please?
looking at what @PatrickLang wrote in the issue i think is pretty important missing info from the docs which should get in asap.

**Comment by derekmahar** [comment]
> /language en
> 
> @derekmahar any chance you can address the comments please?
> looking at what @PatrickLang wrote in the issue i think is pretty important missing info from the docs which should get in asap.

I think you may be confusing me with someone else.  I haven't contributed any code or documentation to this project.

File: docs/tasks/administer-cluster/pids.md (hunk 1 above)
Comment (PatrickLang) [reply docs/tasks/administer-cluster/pids.md]: Is there a special meaning for `0` too? is 0 == no limit?

**Comment by Rajakavitha1** [comment]
@derekwaynecarr any chance you can address the comments please?
looking at what @PatrickLang wrote in the issue i think is pretty important missing info from the docs which should get in asap.

**Comment by DanyC97** [comment]
> > /language en
> > @derekmahar any chance you can address the comments please?
> > looking at what @PatrickLang wrote in the issue i think is pretty important missing info from the docs which should get in asap.
> 
> I think you may be confusing me with someone else. I haven't contributed any code or documentation to this project.

sorry, made a typo in tagging the right person.

**Comment by bradtopol** [comment]
Hi @derekwaynecarr will there be any progress on this soon? Otherwise I plan to close this. you can always let us know and we can reopen. But if I dont hear from you soon Im closing it.  Thanks

**Comment by kbhawkey** [comment]
Hi, @derekwaynecarr. Closing this PR due to inactivity. Feel free to reopen.
/close

**Pull request closed without merging**